### PR TITLE
bump pagy version & bundle update

### LINKED ISSUE #122: Relax Pagy dependencies
@excid3  Pagy has released a lot last month https://rubygems.org/gems/pagy/versions, would you like to relax dependencies.
https://github.com/excid3/madmin/blob/5ef1dfd0c4fbed5abfb606c610533bb05b846537/madmin.gemspec#L22

### PR DESCRIPTION
**Pull request by jitingcn**
Tested on my site, pagy 5.0 update seems did not affect madmin's behaviour.

close #122

### CODE REVIEW

File: madmin.gemspec [hunk 1]
```diff
@@ -19,5 +19,5 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.5.0"
 
   spec.add_dependency "rails", ">= 6.0.3"
-  spec.add_dependency "pagy", ">= 3.5", "< 5.0"
+  spec.add_dependency "pagy", ">= 5.0", "< 6.0"
```
Comment (excid3) [inline madmin.gemspec]: Is there a reason we need to bump the minimum version? The lower we can allow it makes it more compatible with apps that use an older pagy.

Comment (jitingcn) [reply madmin.gemspec]: Right... I didn't take that into account.

Comment (excid3) [reply madmin.gemspec]: That's alright! Sometimes there are breaking changes so I was just wondering. 👍

**Comment by excid3** [comment]
Thanks @jitingcn 👍

**Pull request merged**